kouncil-frontend/apps/kouncil/src/app/sidebar/sidebar.service.ts

```diff
@@ -0,0 +1,29 @@
+import {Injectable} from '@angular/core';
+import {BehaviorSubject, Observable} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SidebarService {
+
+  currentState: string = 'opened';
+
+  private opened$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(this.checkIfOpened());
+
+  changeState(): void {
+    if (this.currentState === 'opened') {
```

Comment:
Maybe it's better to create a dedicated enum for opened/closed values? SidebarState or something